src/forsyde/ut_process_constructors.hpp

```diff
@@ -1741,6 +1741,85 @@ class fanout : public ut_process
 #endif
 };
 
+<<<<<<< HEAD
```

Comment:
It looks like extra lines are added as a result of git merge. Please review and test before continuing with the pull request